docs/data-sources/vault_cluster.md

```diff
@@ -31,6 +31,7 @@ data "hcp_vault_cluster" "example" {
 If not specified, the project specified in the HCP Provider config block will be used, if configured.
 If a project is not configured in the HCP Provider config block, the oldest project in the organization will be used.
 - `timeouts` (Block, Optional) (see [below for nested schema](#nestedblock--timeouts))
+- `vault_plugin` (Block List) The external plugins to install on the vault cluster (see [below for nested schema](#nestedblock--vault_plugin))
```

Comment:
I have `vault_plugin` because it is specified as an individual resource.

The implementation would look like this in pseduo-terraform

```
resource `hcp_vault_cluster` {
       vault_plugin {
		plugin_type = "SECRET"
		plugin_name = "plugin1"
	}
	vault_plugin {
		plugin_type = "DATABASE"
		plugin_name = "plugin2"
	}
}
```

since each plugin is specified is a separate `vault_plugin` block, that is why I have singular `vault_plugin`

Reply:
thanks for explaining